src/parser.ts

```diff
@@ -3912,54 +3912,54 @@ export class Parser extends DiagnosticEmitter {
         return Node.createObjectLiteralExpression(names, values, tn.range(startPos, tn.pos));
       }
       // AssertionExpression (unary prefix)
-      case Token.LESSTHAN: {
+      case Token.LessThan: {
         let toType = this.parseType(tn);
         if (!toType) return null;
-        if (!tn.skip(Token.GREATERTHAN)) {
+        if (!tn.skip(Token.GreaterThan)) {
           this.error(
             DiagnosticCode._0_expected,
             tn.range(), ">"
           );
           return null;
         }
-        let expr = this.parseExpression(tn, Precedence.CALL);
+        let expr = this.parseExpression(tn, Precedence.Call);
         if (!expr) return null;
         return Node.createAssertionExpression(
-          AssertionKind.PREFIX,
+          AssertionKind.Prefix,
           expr,
           toType,
           tn.range(startPos, tn.pos)
         );
       }
-      case Token.IDENTIFIER: {
+      case Token.Identifier: {
         let identifierText = tn.readIdentifier();
         if (identifierText == "null") return Node.createNullExpression(tn.range()); // special
         let identifier = Node.createIdentifierExpression(identifierText, tn.range(startPos, tn.pos));
-        if (tn.skip(Token.TEMPLATELITERAL)) {
+        if (tn.skip(Token.TemplateLiteral)) {
           return this.parseTemplateLiteral(tn, identifier);
         }
-        if (tn.peek(true) == Token.EQUALS_GREATERTHAN && !tn.nextTokenOnNewLine) {
+        if (tn.peek(true) == Token.EqualsGreaterThan && !tn.nextTokenOnNewLine) {
```

Comment:
`EqualsGreaterThan`, and similar combined tokens, are a little strange. In `EQUALS_GREATERTHAN`, it was clear that this is `=` + `>`. Would it perhaps make sense to use `Greaterthan`, as if it was a single word?

Reply:
`Greaterthan` is really hard for readability compare to `GreaterThan`. Especially for compound variants. Just compare:
```ts
GREATERTHAN_GREATERTHAN_GREATERTHAN_EQUALS
GreaterthanGreaterthanGreaterthanEquals
GreaterThanGreaterThanGreaterThanEquals
```